docs/maui/views/MediaElement.md

```diff
@@ -76,11 +76,65 @@ public class MainActivity : MauiAppCompatActivity
 }
 ```
 
+Edit the `androidManifest.xml` manifest file and add the following.
+
+```csharp
+<application android:allowBackup="true" android:icon="@mipmap/appicon" android:enableOnBackInvokedCallback="true" android:supportsRtl="true">
+    <service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
+      <intent-filter>
+        <action android:name="android.intent.action.MEDIA_BUTTON" />
+      </intent-filter>
+      <intent-filter>
+        <action android:name="androidx.media3.session.MediaSessionService"/>
+      </intent-filter>
+    </service>
+</application>
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
+<uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
+```
+
+> [!NOTE]
+> This modification to the Android manifest enables metadata display when playing a video. It provides support for notifications and is essential
+ for notifications to function across all relevant APIs. The change introduces a service and grants necessary permissions.
```

Comment:
This is an open question and I can't decide what the correct answer is but should we have separate sections; one to detail how to initialise media playback and one to add metadata support?

One the one hand it feels like it could be smaller and more concise sections if we separate them but then if a developer wants both it Is spread across two sections. Thoughts?

Reply:
Feel free to make changes and edit as you want. I started this PR to get the ball rolling.